ga4gh/datamodel/sequenceAnnotations.py

```diff
@@ -158,14 +158,26 @@ class AbstractFeatureSet(datamodel.DatamodelObject):
     """
     compoundIdClass = datamodel.FeatureSetCompoundId
 
-    def __init__(
-            self, parentContainer, localId, referenceSetId=None):
+    def __init__(self, parentContainer, localId):
         super(AbstractFeatureSet, self).__init__(parentContainer, localId)
-        self._referenceSetId = referenceSetId
```

Comment:
Because in VariantSets and ReadGroupSets we need the actual reference objects to look things up. I opted the object reference here over the ID for uniformity.